### PR TITLE
Fix #8737: Added title tag for tooltip information.

### LINKED ISSUE #8737: Avatar: Missing title tag for tooltip information.
**Description**
Missing title tag for tooltip information.

**Example XHTML**
```xml
<p:avatar label="Johann Chrysostom Wolfgang Amadeus Mozart" title="Wolfgang Amadeus Mozart"/>
```

### PR DESCRIPTION
**Pull request by aripddev**

### CODE REVIEW

**Review by jepsar — CHANGES_REQUESTED** [review]
Can you please update the documentation (in the parent project)?

**Comment by jepsar** [comment]
I think the title will always be the same as the label. Do we want to explicitly set the title while it's the same as the label?

Maybe we can use "true" to copy the label value?

**Comment by melloware** [comment]
I definitely like the flexibility of declaring a different title but maybe we should just default it to label and let the user override it to "" if they truly don't want a title?

**Comment by aripddev** [comment]
Is it enough to set it to `label` instead of setting it to null?

```java
public String getTitle() {
   return (String) getStateHelper().eval(PropertyKeys.title, getLabel());
}
```

**Comment by aripddev** [comment]
> Is it enough to set it to `label` instead of setting it to null?
> 
> ```java
> public String getTitle() {
>    return (String) getStateHelper().eval(PropertyKeys.title, getLabel());
> }
> ```

yeap, that is worked.

File: primefaces/src/main/java/org/primefaces/component/avatar/AvatarRenderer.java [hunk 1]
```diff
@@ -74,6 +75,10 @@ public void encodeEnd(FacesContext context, UIComponent component) throws IOExce
             writer.writeAttribute("style", style, "style");
         }
 
+        if (title != null) {
```
Comment (jepsar) [inline primefaces/src/main/java/org/primefaces/component/avatar/AvatarRenderer.java]:
```suggestion
        if (!LangUtils.isEmpty(title)) {
```

**Review by melloware — APPROVED** [review]

**Review by jepsar — APPROVED** [review]

**Comment by jepsar** [comment]
Fixed the doc https://github.com/primefaces/primefaces/commit/dab97cb78f3d571865878d0bf8b2140ede38ac6c

**Pull request merged**